sync/src/relayer/get_transactions_process.rs

```diff
@@ -65,6 +67,15 @@ impl<'a> GetTransactionsProcess<'a> {
                         .transaction(tx.data())
                         .build()
                 })
+                .take_while(|relay_tx| {
+                    // If the size of RelayTransactions response is too large we
+                    // just skip it, since this case is rare and we can get
+                    // missing transactions by BlockTransactions message later.
+                    total_size += relay_tx.as_slice().len();
```

Comment:
may overflow

Reply:
Actually it's not possible, total_size must not greater than `MAX_RELAY_TXS_SIZE`

`total_size <= MAX_RELAY_TXS_SIZE`